srcCpp/PerftestTransport.cxx

```diff
@@ -806,6 +824,14 @@ std::string PerftestTransport::printTransportConfigurationSummary()
         stringStream << "\tNic: " << allowInterfaces << "\n";
     }
 
+    stringStream << "\tUse Multicast: "
```

Comment:
After discussing with @alfonso-soto, only display information about multicast when it is set by command line parameter:
```
    if (useMulticast) {
        if (allowsMulticast()) {
            stringStream << "\tUse Multicast: True";
        } else {
            stringStream << "\t(Multicast is not supported for "
                         << transportConfig.nameString << ")";
        }
    }
```

Reply:
Done.

Reply:
you did not update the code of this change.

Reply:
I think we should always display the Multicast info.